core/src/main/java/pl/agh/edu/GdxGame.java

```diff
@@ -25,6 +26,7 @@ public void create() {
 		LanguageConfig.setLanguage(LanguageConfig.getLanguage());
 		GraphicConfig.setFullscreenMode(GraphicConfig.isFullscreen());
 		GraphicConfig.changeResolution(GraphicConfig.getResolution());
+		Engine engine = new Engine();
```

Comment:
jak już to lepiej żeby było jako pole poza konstruktorem ale czaję fażę że pewnie żeby testować